### ISSUE TITLE
AWS Problem creating stack - 'Only one subnet per availability zone is supported...'

### ISSUE THREAD

**Issue by jeffelman** [openedx-unsupported/configuration#569]
While trying to create a stack on AWS with CloudFormation (using the configuration template edx-server-single-instance.json), there is a create failure with the following error:

AWS::AutoScaling::AutoScalingGroup  MongoServerASGroup      CREATE_FAILED   Only one subnet per availability-zone is supported in each AutoScalingGroup

Any help or suggestions about what's going wrong or how to fix this will be appreciated. Thanks. /Jeff

**Comment by e0d** [comment]
So this would happen if two of the three subnets for mongo ended up in the same AZ and then were associated with the ASG by CloudFormation.  I don't see why this would happen, what region are you in?

**Comment by jeffelman** [comment]
You hit it on the nail. Yes, my error. I was trying to create the stack in the us-west-1, and had gotten the error 

"Value (us-west-1b) for parameter availabilityZone is invalid. Subnets 
can currently only be created in the following availability zones:
us-west-1c, us-west-1a."

So (not so) cleverly, I edited the template file to replace us-west-1b by us-west-1c in the mappings structure, meaning that us-west-1c was being used twice. Clearly a mistake. I changed to us-west-2 and it worked fine. Thanks for the analysis. /Jeff
